build-aux/flatpak/com.github.akiraux.akira.json

```diff
@@ -1,56 +1,120 @@
 {
-   "app-id": "com.github.akiraux.akiraDevel",
-   "runtime": "org.gnome.Platform",
-   "runtime-version": "3.38",
-   "sdk": "org.gnome.Sdk",
-   "base":"io.elementary.BaseApp",
-   "base-version": "juno-20.08",
-   "command": "com.github.akiraux.akiraDevel",
-   "cleanup": [
-      "/include",
-      "/lib/pkgconfig",
-      "/lib/debug",
-      "/share/vala",
-      "/man",
-      "*.a",
-      "*.la"
-   ],
-   "finish-args": [
-      "--share=ipc",
-      "--socket=fallback-x11",
-      "--socket=wayland",
-      "--talk-name=org.gtk.vfs.*"
-   ],
-   "modules": [
-      {
-         "name": "goocanvas",
-         "config-opts": ["--enable-python=no"],
-         "build-options" : {
-             "make-install-args" : [
-                 "girdir=/app/share/gir-1.0",
-                 "typelibdir=/app/lib/girepository-1.0"
-             ]
-         },
-         "sources": [
-            {
-               "type": "git",
-               "url": "https://gitlab.gnome.org/GNOME/goocanvas",
-               "branch": "goocanvas-3.0"
-            }
-         ]
+  "app-id": "com.github.akiraux.akiraDevel",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "40",
+  "sdk": "org.gnome.Sdk",
+  "base": "io.elementary.BaseApp",
+  "base-version": "juno-20.08",
+  "command": "com.github.akiraux.akiraDevel",
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/lib/debug",
+    "/share/vala",
+    "/man",
+    "*.a",
+    "*.la"
+  ],
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--talk-name=org.gtk.vfs.*",
+    "--filesystem=xdg-run/gvfs",
+    "--filesystem=xdg-run/gvfsd"
```

Comment:
Those were always needed to be able to open file from another hard drive, remote mounts...